config/log4j2.yaml

```diff
@@ -126,18 +126,12 @@ Configuration:
         additivity: false
         AppenderRef:
           ref: RequestAppender
-      # KRaft mode controller logger
+      # Controller logger
       - name: org.apache.kafka.controller
         level: INFO
         additivity: false
         AppenderRef:
           ref: ControllerAppender
-      # ZK mode controller logger
-      - name: kafka.controller
```

Comment:
Perhaps we can also clean up the `log4j2.yaml` configuration for our e2e, given that the "kafka.controller" package is being completely removed in 4.0.

https://github.com/apache/kafka/blob/trunk/tests/kafkatest/services/kafka/templates/log4j2.yaml#L250

Reply:
I would prefer to remove it when we clean the system tests.